python_ta/typecheck/base.py

```diff
@@ -149,10 +149,50 @@ class TupleSubscript(TypeVar, _root=True):
     pass
 
 
-def create_Callable(args: Iterable[type], rtype, poly_vars=None):
-    poly_vars = poly_vars or []
+_TYPESHED_TVARS = {
+    '_T': TypeVar('_T'),
+    '_T_co': TypeVar('_T_co', covariant=True),
+    '_KT': TypeVar('_KT'),
+    '_VT': TypeVar('_VT'),
+    '_S': TypeVar('_S'),
+    '_T1': TypeVar('_T1'),
+    '_T2': TypeVar('_T2'),
+    '_T3': TypeVar('_T3'),
+    '_T4': TypeVar('_T4'),
+    '_T5': TypeVar('_T5'),
+    '_TT': TypeVar('_TT', bound='type'),
+    'function': Callable[[Any], Any]
+}
+
+
+_BUILTIN_TO_TYPING = {
+    'list': 'List',
+    'dict': 'Dict',
+    'tuple': 'Tuple',
+    'set': 'Set',
+    'frozenset': 'FrozenSet',
+    'function': 'Callable'
+}
+
```

Comment:
I've moved these dictionaries here because they're used in `get_poly_vars`, which in turn is called in the `create_Callable` calls in `TYPE_SIGNATURES`